lib/generated/intl/messages_en.dart

```diff
@@ -19,6 +19,402 @@ typedef String MessageIfAbsent(String messageStr, List<dynamic> args);
+        "polkadexIfYouUninstall": MessageLookupByLibrary.simpleMessage(
+            "Polkadex does not store any data , this is just a temporary data, if you uninstall the Polkadex App your name and referral ID will be removed."),
+        "polkadexInterfaceTour":
+            MessageLookupByLibrary.simpleMessage("Polkadex interface tour"),
+        "polkadexLanguage":
+            MessageLookupByLibrary.simpleMessage("Polkadex Language"),
+        "polkadexOnlyEnable": MessageLookupByLibrary.simpleMessage(
+            "Polkadex does not store any data , by security, Polkadex only enable withdrawals by confirmation via Two-Factor Authentication (2FA)."),
+        "polkadexStatus":
+            MessageLookupByLibrary.simpleMessage("Polkadex Status"),
+        "polkadexSystemMessage":
+            MessageLookupByLibrary.simpleMessage("Polkadex system message"),
+        "previous": MessageLookupByLibrary.simpleMessage("Previous"),
+        "price": MessageLookupByLibrary.simpleMessage("Price"),
+        "priceCoin": m2,
+        "priceShouldBeGreater": MessageLookupByLibrary.simpleMessage(
+            "Price should be greater than 0.00"),
+        "priceShouldBeLess": MessageLookupByLibrary.simpleMessage(
+            "Price should be less than wallet balance"),
+        "pricingLength": MessageLookupByLibrary.simpleMessage("Pricing Length"),
+        "privacyAndSecurity":
+            MessageLookupByLibrary.simpleMessage("Privacy & Security"),
+        "privacyPolicy": MessageLookupByLibrary.simpleMessage("Privacy Policy"),
+        "purchaseAddedToOpen": MessageLookupByLibrary.simpleMessage(
+            "Purchase added to open orders"),
+        "push": MessageLookupByLibrary.simpleMessage("Push"),
+        "rank": MessageLookupByLibrary.simpleMessage("Rank"),
+        "rankingList": MessageLookupByLibrary.simpleMessage("Ranking List"),
+        "recent": MessageLookupByLibrary.simpleMessage("Recent"),
+        "reddit": MessageLookupByLibrary.simpleMessage("Reddit"),
+        "referral": MessageLookupByLibrary.simpleMessage("Referral"),
+        "removalOfLinksFrom": MessageLookupByLibrary.simpleMessage(
+            "Removal of links from our website"),
+        "repeatPassword":
+            MessageLookupByLibrary.simpleMessage("Repeat Password"),
+        "repeatYourPassword":
+            MessageLookupByLibrary.simpleMessage("Repeat your password"),
+        "reservationOfRights":
+            MessageLookupByLibrary.simpleMessage("Reservation of Rights"),
+        "restoreExistingWallet":
+            MessageLookupByLibrary.simpleMessage("Restore existing wallet"),
+        "save": MessageLookupByLibrary.simpleMessage("Save"),
+        "scanOrShare": MessageLookupByLibrary.simpleMessage("Scan or share"),
+        "searchNameOrTicket":
+            MessageLookupByLibrary.simpleMessage("Search name or ticket"),
+        "secureWithFingerPrint":
+            MessageLookupByLibrary.simpleMessage("Secure with FingerPrint"),
```

Comment:
```suggestion
            MessageLookupByLibrary.simpleMessage("Secure with Biometric login"),
```

The api supports faceid and iris too

Reply:
The flutter plugin i'm planning to use supports biometric and faceid: https://pub.dev/packages/local_auth

Reply:
Yes so the message should be inline with that. Does `local_auth` have secure storage too? I think I saw another way it should be done so please double check before coding too deep.